dockerfiles/init/modules/che-kubernetes-helm/tiller-rbac.yaml

```diff
@@ -0,0 +1,28 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: tiller-role-binding
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Why do we need to add the service account `tiller`? And why should it have `cluster-admin` role?

Reply:
Basically, it's because tiller needs to create Kubernetes resources in namespace other than the namespace in which tiller is installed. [Here](https://github.com/kubernetes/helm/blob/master/docs/rbac.md#tiller-and-role-based-access-control) are more details.